test/e2e/scripts/auth_utils.sh

```
@@ -537,15 +584,22 @@ EOF
   _append "This summary helps compare local vs CI runs:"
   _append ""
   local total_models total_subs total_authpolicies total_kuadrant_authpolicies
+  local total_aitenants total_inference_models total_inference_providers
   total_models=$(echo "$models_json" | jq '. | length' 2>/dev/null || echo "0")
   total_subs=$(echo "$subscriptions_json" | jq '. | length' 2>/dev/null || echo "0")
   total_authpolicies=$(kubectl get maasauthpolicies -n $MAAS_SUBSCRIPTION_NAMESPACE -o json 2>/dev/null | jq -r '.items | length' 2>/dev/null || echo "0")
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)